LayoutKit.playground/Sources/UIControl+Handler.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+private var controlHandlerKey: Int8 = 0
+
+extension UIControl {
+
+    public func addHandler(for controlEvents: UIControlEvents, handler: @escaping (UIControl) -> ()) {
+        let target = CocoaTarget<UIControl>(handler)
+        objc_setAssociatedObject(self, &controlHandlerKey, target, .OBJC_ASSOCIATION_RETAIN)
+
+        self.addTarget(target, action: #selector(target.sendNext), for: controlEvents)
```

Comment:
Ideally this code would make sure to remove the previous target, if there was a CocoaTarget object associated with self before the new one created here, right? If so, it'd be good to have this sample code illustrate that, or at least point out the issue in a comment.